Dockerfiles/agent/README.md

```diff
@@ -36,10 +36,6 @@ environment variables:
 - `DD_PROXY_HTTPS`: an http URL to use as a proxy for `https` requests.
 - `DD_PROXY_NO_PROXY`: a space-separated list of URLs for which no proxy should be used.
 
-Note: At the moment, the trace-agent only supports setting a proxy in `datadog.yaml`, and does not
```

Comment:
Since the trace-agent only supports the env starting with 6.5.0, I think we should make this clear here :)